auth0_flutter/android/src/test/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandlerTest.kt

```diff
@@ -33,12 +34,14 @@ class CredentialsManagerMethodCallHandlerTest {
         arguments: HashMap<String, Any?> = defaultArguments,
         requestHandlers: List<CredentialsManagerRequestHandler>,
         activity: Activity? = null,
```

Comment:
Is this still needed?

Reply:
As mentioned in the previous comment, we still use it in `requireAuthentication` so it is required in tests as well